### PR TITLE
docs: Non-serializable Properties - edit pass

### PR DESCRIPTION
**Pull request by craigshoemaker**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests

# Description

I hope these edits didn't change the original meaning 😉 

# Use cases and why

n/a

# Checklist:

- [x] My code follows the [developer guidelines of this project](../CONTRIBUTING.md)
- [x]  I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/pages/tutorial/store/no-serialize/index.mdx [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: Non-serializable Properties
 layout: tutorial
 ---
 
-Stores must always be serializable. However, there are times when it is useful to store a value on a store that is not serializable. Such a value will persist only while the application is resumed. If the application is paused the value will be thrown away. (This is not an idiomatic way to do this, only a contrived example.)
+Stores must always be serializable, however sometimes you may want persist a value on a store that can't be serialized. Values like these persist in the browser only when the application is resumed on the client.
```
Comment (shairez) [inline packages/docs/src/pages/tutorial/store/no-serialize/index.mdx:6]: missing "to" after "you may want"

File: packages/docs/src/pages/tutorial/store/no-serialize/index.mdx [hunk 2]
```diff
@@ -3,6 +3,8 @@ title: Non-serializable Properties
 layout: tutorial
 ---
 
-Stores must always be serializable. However, there are times when it is useful to store a value on a store that is not serializable. Such a value will persist only while the application is resumed. If the application is paused the value will be thrown away. (This is not an idiomatic way to do this, only a contrived example.)
+Stores must always be serializable, however sometimes you may want persist a value on a store that can't be serialized. Values like these persist in the browser only when the application is resumed on the client. 
 
-In this example when the clock is started we need to save a cleanup function in `store.cleanup` so that the `stop` button can clear the interval.
\ No newline at end of file
+When the application is paused, and execution returns to the server, non-serializable properties are discarded.
```
Comment (shairez) [inline packages/docs/src/pages/tutorial/store/no-serialize/index.mdx:8]: What do you mean by "execusion returns to the server"? 

AFAIK, execusion starts on the server and then getting "paused" (serialized) and then resumes on the client.
Another server call is either ajax or a full page load (which will not continue, just start over from a different entry point)

@mhevery can you verify that what I've written here is correct?

File: packages/docs/src/pages/tutorial/store/no-serialize/index.mdx (hunk 1 above)
Comment (craigshoemaker) [inline packages/docs/src/pages/tutorial/store/no-serialize/index.mdx:6]:
```suggestion
Stores must always be serializable, however sometimes you may to want persist a value on a store that can't be serialized. Values like these persist in the browser only when the application is resumed on the client. 
```

Comment (craigshoemaker) [reply packages/docs/src/pages/tutorial/store/no-serialize/index.mdx:6]: thanks!

File: packages/docs/src/pages/tutorial/store/no-serialize/index.mdx (hunk 2 above)
Comment (craigshoemaker) [reply packages/docs/src/pages/tutorial/store/no-serialize/index.mdx:8]: That makes sense. Thanks for clarifying my thinking here. I'll wait for Misko to chime in, but I can certainly adjust accordingly.

**Review by mhevery — APPROVED** [review]

**Comment by mhevery** [comment]
Holding merge until https://github.com/BuilderIO/qwik/pull/834 is in

File: packages/docs/src/pages/tutorial/store/no-serialize/index.mdx (hunk 2 above)
Comment (shairez) [reply packages/docs/src/pages/tutorial/store/no-serialize/index.mdx:8]: @mhevery did you see this? :)

Comment (mhevery) [reply packages/docs/src/pages/tutorial/store/no-serialize/index.mdx:8]: `and execution returns to the server` does not make sense. I think you want to write:
```
When the application is paused, non-serializable properties are discarded.
```

File: packages/docs/src/pages/tutorial/store/no-serialize/index.mdx [hunk 3]
```diff
@@ -3,6 +3,8 @@ title: Non-serializable Properties
 layout: tutorial
 ---
 
-Stores must always be serializable. However, there are times when it is useful to store a value on a store that is not serializable. Such a value will persist only while the application is resumed. If the application is paused the value will be thrown away. (This is not an idiomatic way to do this, only a contrived example.)
+Stores must always be serializable, however sometimes you may to want persist a value on a store that can't be serialized. Values like these persist in the browser only when the application is resumed on the client. 
 
-In this example when the clock is started we need to save a cleanup function in `store.cleanup` so that the `stop` button can clear the interval.
\ No newline at end of file
+When the application is paused, and execution returns to the server, non-serializable properties are discarded.
```
Comment (craigshoemaker) [inline packages/docs/src/pages/tutorial/store/no-serialize/index.mdx:8]:
```suggestion
When the application is paused, non-serializable properties are discarded.
```

File: packages/docs/src/pages/tutorial/store/no-serialize/index.mdx (hunk 2 above)
Comment (craigshoemaker) [reply packages/docs/src/pages/tutorial/store/no-serialize/index.mdx]: This is now fixed!

**Comment by adamdbradley** [comment]
Thanks, updated in this PR: https://github.com/BuilderIO/qwik/pull/888

**Pull request closed without merging**